src/mjolnir/graphbuilder.cc

```diff
@@ -1333,6 +1334,7 @@ void BuildLocalTiles(const unsigned int thread_count,
                      const std::string& tile_dir,
                      DataQuality& stats,
                      const boost::property_tree::ptree& pt) {
+  SCOPED_TIMER();
```

Comment:
why do we need this, `Build` calls this function so they are measuring mostly the same thing